app/config/settings.py

```diff
@@ -1144,6 +1144,10 @@
         "task": "grandchallenge.challenges.tasks.update_challenge_results_cache",
         "schedule": crontab(minute="*/5"),
     },
+    "update_challenge_cost_statistics": {
+        "task": "grandchallenge.challenges.tasks.update_challenge_cost_statistics",
+        "schedule": crontab(minute=0, hour=0),
```

Comment:
Could you put this in time order for the dictionary? There is another task running at 00:00, so maybe move this to the next free slot at 05:00.